Doc/whatsnew/3.14.rst

```diff
@@ -1204,6 +1204,17 @@ Other language changes
 
   .. _Jython: https://www.jython.org/
 
+* The :func:`int` built-in no longer delegates to :meth:`~object.__trunc__`.
+  Classes that want to support conversion to :func:`!int` must implement
+  either :meth:`~object.__int__` or :meth:`~object.__index__`.
+  (Contributed by Mark Dickinson in :gh:`119743`.)
+
+* Using :data:`NotImplemented` in a Boolean context
```

Comment:
Or let's keep the lower case per https://github.com/python/docs-community/issues/129.
```suggestion
* Using :data:`NotImplemented` in a boolean context
```